test/framework/integration_framework/fake_peer/block_storage.hpp

```diff
@@ -25,33 +26,28 @@ namespace integration_framework {
 
     class BlockStorage final {
      public:
-      using BlockPtr = std::shared_ptr<shared_model::proto::Block>;
+      using BlockPtr = std::shared_ptr<const shared_model::proto::Block>;
       using HeightType = shared_model::interface::types::HeightType;
       using HashType = shared_model::crypto::Hash;
 
+      BlockStorage();
+      BlockStorage(const BlockStorage &);
```

Comment:
Are these constructors required to be defined explicitly?

Reply:
That is because the mutex class member, which cannot be copied.